content/en/docs/protocol/aqua-name-resolution.md

```diff
@@ -13,4 +13,4 @@ The following hashes are resolved by the ANS:
 * genesis_hashes to titles.
 * domain_ids to registered endpoints of that domain.
 
-See the [reference implementation](https://github.com/inblockio/aqua-verifier-webextension/blob/main/src/name_resolution.tsx).
+See the [reference implementation](https://github.com/inblockio/aqua-verifier-webextension/blob/main/src/name_resolution_configuration.tsx).
```

Comment:
This needs to be https://github.com/inblockio/aqua-verifier-webextension/blob/main/src/name_resolver.ts.

Reply:
Updated it